exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 var studentCount = 16;
```

Comment:
Actually we need to use variables names for all of these groups like below.

var moreStudentsThanMentors = studentCount > mentorCount; 

var enoughSpaceInRoom = roomMaxCapacity >= 25; 

var sameName = personA === personB;